src/components/FormPlanning/index.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+
+import { Box, TextField, Select, InputLabel, MenuItem, FormControl, Button } from '@mui/material';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import TimePicker from '@mui/lab/TimePicker';
+
+export const FormPlanning = () => {
+    const [date, setDate] = useState(null);
+    const [hour, setHour] = useState(null);
+    const [type, setType] = useState(null);
+    const [platform, setPlatform] = useState(null);
+
+    const handleChange = (event) => {
+        setType(event.target.value);
+    };
+
+    const changePlatform = (event) => {
+        setPlatform(event.target.value);
+    };
+    return (
+        <Box component="form">
+            <TextField id="outlined-basic" label="Interview title" variant="outlined" sx={{ width: '100%', mb: 2 }} />
+            <Box sx={{ mb: 2 }}>
+                <LocalizationProvider dateAdapter={AdapterDateFns} sx={{ mr: 1 }}>
+                    <DatePicker
+                        label="Choose date"
+                        value={date}
+                        onChange={(newValue) => {
+                            setDate(newValue);
+                        }}
+                        renderInput={(params) => <TextField {...params} />}
+                    />
+                </LocalizationProvider>
+                <LocalizationProvider dateAdapter={AdapterDateFns} sx={{ mr: 1 }}>
+                    <TimePicker
+                        label="Choose hour"
+                        value={hour}
+                        onChange={(newValue) => {
+                            setHour(newValue);
+                        }}
+                        renderInput={(params) => <TextField {...params} />}
+                    />
+                </LocalizationProvider>
+                <FormControl sx={{ width: 200 }}>
+                    <InputLabel id="demo-simple-select-label">Interview type</InputLabel>
+                    <Select
+                        labelId="demo-simple-select-label"
+                        id="demo-simple-select"
+                        value={type}
+                        label="Interview type"
+                        onChange={handleChange}
+                    >
+                        <MenuItem value="remote">Remote</MenuItem>
+                        <MenuItem value="presencial">Presencial</MenuItem>

```

Comment:
Sounds better **Face to Face**. What do you think @odagora ? 🤔

Reply:
Fixed!